src/platform/cannonlake/include/platform/memory.h

```diff
@@ -338,11 +338,8 @@
 	HEAP_RT_LP_COUNT128 * 128 + HEAP_RT_LP_COUNT256 * 256 + \
 	HEAP_RT_LP_COUNT512 * 512 + HEAP_RT_LP_COUNT1024 * 1024)
 
-#define HEAP_LP_BUFFER_BASE \
-	(HEAP_LP_RUNTIME_BASE + HEAP_LP_RUNTIME_SIZE)
-#define HEAP_LP_BUFFER_SIZE \
-	(LP_SRAM_SIZE - HEAP_LP_RUNTIME_SIZE - SOF_LP_STACK_SIZE - \
-	HEAP_LP_SYSTEM_SIZE)
+#define HEAP_LP_BUFFER_BASE LP_SRAM_BASE
```

Comment:
With this change you will have whole LP_SRAM for heap buffer but also in the same place there will be heaps like HEAP_LP_SYSTEM_BASE or HEAP_LP_RUNTIME_BASE, change them if not needed

Reply:
@abonislawski, and this is the intention we need whole LPSRAM for heap buffer.

Reply:
@mrajwa I mean in this case not needed defines like HEAP_LP_RUNTIME_BASE should be deleted to make this clear